### PR TITLE
Property and history cleanup

### PR DESCRIPTION
**Pull request by stevenger**
# Improved the ssh connection error message from "Failed to run the following command: test -e ~/.jwala" to "SSH connection to <host>:<port> for user <user> failed. Check your host, port, user and password."
    # Removed references to remote.jwala.java.home and PropertyKeys.REMOTE_JAVA_HOME. This should now all be derived from the JVM.
    # Created classes that extend ControlJvmRequest to encapsulate the history messages.
    # Renamed DEPLOY_CONFIG_ARCHIVE to DEPLOY_JVM_ARCHIVE.
    # Added more detail to history message logging for different JVM operations. Still needed for the web server.
    # Removed deprecated @Value("${spring.messaging.topic.serverStates:/topic/server-states}") from JvmControlServiceImpl.
    # Removed validation check for required setenv template file because it's no longer needed. Java home is set when creating the service rather than in the setenv file.
    # Temporarily commenting out the "Expecting ContextRefreshedEvent. Skipping." log message until the logic for this class is added.
    # Refactoring to remove the Aem acronym from AemSsh.
    # Moved ssh properties from AemSshProperty class to PropertyKeys class.
    # Changed DISTRIBUTE_JDK history message to only be written when JDK is distributed rather than every time Jwala checks for a remote JDK. Modified the history text to read Installing JDK  on host "...".
    # Removed unused properties - remote.paths.tomcat.core and jwala.tomcat.service.exe, h2.tcp.server.params, h2.web.server.params, remote.jvm.account.username, remote.jvm.account.password.encrypted.
    # Renamed AemSsh properties to ssh. Removed AemSsh.privateKeyFile=#REQUIRED#, AemSsh.knownHostsFile=#REQUIRED# and AemSsh.port=#REQUIRED" from vars.properties because they're not required.
    # Changed SshConfiguration to require username and password only (removed ssh port). Ssh port now defaults to 22 but can be overridden in vars.properties.

### CODE REVIEW

**Review by jmahmoodcc — APPROVED** [review]

File: jwala-common/src/main/java/com/cerner/jwala/common/request/jvm/StartServiceControlJvmRequest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * Created by sg038069 on 4/9/17.
+ */
+public class StartServiceControlJvmRequest  extends ControlJvmRequest {
+
+    private final Jvm jvm;
+
+    public StartServiceControlJvmRequest(final Jvm jvm) {
+        super(jvm.getId(), JvmControlOperation.START);
+        this.jvm = jvm;
+        System.out.println("################## jvm = " + jvm);
```
Comment (jeddcuison) [inline jwala-common/src/main/java/com/cerner/jwala/common/request/jvm/StartServiceControlJvmRequest.java]: Remove

File: jwala-services/src/test/java/com/cerner/jwala/service/jvm/impl/JvmServiceImplTest.java [hunk 2]
```diff
@@ -630,9 +615,14 @@ public void testGenerateAndDeployConfig() throws CommandFailureException, IOExce
         when(Config.mockJvmControlService.executeCreateDirectoryCommand(any(Jvm.class), anyString())).thenReturn(commandOutput);
         when(Config.mockJvmControlService.executeChangeFileModeCommand(any(Jvm.class), anyString(), anyString(), anyString())).thenReturn(commandOutput);
 
-        when(Config.mockJvmControlService.controlJvm(eq(new ControlJvmRequest(mockJvm.getId(), JvmControlOperation.DELETE_SERVICE)), any(User.class))).thenReturn(commandOutput);
-        when(Config.mockJvmControlService.controlJvm(eq(new ControlJvmRequest(mockJvm.getId(), JvmControlOperation.DEPLOY_CONFIG_ARCHIVE)), any(User.class))).thenReturn(commandOutput);
-        when(Config.mockJvmControlService.controlJvm(eq(new ControlJvmRequest(mockJvm.getId(), JvmControlOperation.INSTALL_SERVICE)), any(User.class))).thenReturn(commandOutput);
+//        when(Config.mockJvmControlService.controlJvm(eq(new DeleteServiceControlJvmRequest(mockJvm)), any(User.class))).thenReturn(commandOutput);
```
Comment (jeddcuison) [inline jwala-services/src/test/java/com/cerner/jwala/service/jvm/impl/JvmServiceImplTest.java]: Remove commented out codes

File: jwala-common/src/main/java/com/cerner/jwala/common/request/jvm/StartServiceControlJvmRequest.java (hunk 1 above)
Comment (jeddcuison) [reply jwala-common/src/main/java/com/cerner/jwala/common/request/jvm/StartServiceControlJvmRequest.java]: DONE

File: jwala-services/src/test/java/com/cerner/jwala/service/jvm/impl/JvmServiceImplTest.java (hunk 2 above)
Comment (jeddcuison) [reply jwala-services/src/test/java/com/cerner/jwala/service/jvm/impl/JvmServiceImplTest.java]: DONE

**Review by jeddcuison — APPROVED** [review]

**Pull request merged**